### ISSUE TITLE
[ENHANCEMENT]Remove Edelman response function

### ISSUE THREAD

**Issue by mbakker7** [pastas/pastas#475]
The Edelman response function has been in the code for a while, but was never operational as it was not imported upon regular import of `pastas`. 

The Edelman response function has problems, as it takes infinitely long before the step response reaches 1, and as a result it takes very long for it to reach the cutoff. Almost the same response can be obtained with the Polder function, which reaches 1 within a finite time.

I propose to remove the Edelman response function from the code base.

**Comment by rubencalje** [comment]
The Edelman function was proposed by Wilbert Berendrecht in 2019 (see your mail at the time), after which I committed it in Pastas. I experimented with it a bit at the time. The function is (or was) operational, but you can only reach it by `pastas.rfunc.Edelman`, and not directly by `pastas.Edelman`.

Wilbert maximized tmax at 20000, and he preferred it over the Polder-function, because the parameters of the Polder function were too correlated. Unless you think this is not true, or Edelman causes problems keeping the rest of pastas up to date, I am in favor of keeping it.

**Comment by dbrakenhoff** [comment]
Move to plugins when plugins API is complete.

**Comment by raoulcollenteur** [comment]
I agree, @martinvonk make a deprecationwarning for new release.

**Comment by martinvonk** [comment]
sudo make me a sandwich

**Comment by raoulcollenteur** [comment]
Any progress on this @martinvonk ? Would be required at the next release

**Comment by raoulcollenteur** [comment]
@martinvonk can you check the status of this issue and close if appropriate?

**Comment by martinvonk** [comment]
Yes this can be closed. Code was fully removed in #1173. The Edelman rfunc is still available in Pastas-Plugins
